api/v1alpha1/backupstorage_types.go

```diff
@@ -22,22 +22,26 @@ import (
 const (
 	// BackupStorageTypeS3 is a type of S3 object storage.
 	BackupStorageTypeS3 BackupStorageType = "s3"
+	// BackupStorageTypeAzure is a type of azure blob storage.
+	BackupStorageTypeAzure BackupStorageType = "azure"
+	// BackupStorageTypeGCS is a type of Google Cloud Storage object storage.
+	BackupStorageTypeGCS BackupStorageType = "gcs"
 )
 
 // BackupStorageType is a type of backup storage.
 type BackupStorageType string
 
 // BackupStorageSpec defines the desired state of BackupStorage.
 type BackupStorageSpec struct {
-	// Type is a type of backup storage. Currently only S3 is supported.
-	// +kubebuilder:validation:Enum=s3
+	// Type is a type of backup storage.
+	// +kubebuilder:validation:Enum=s3;gcs;azure
 	Type BackupStorageType `json:"type"`
 	// Bucket is a name of bucket.
 	Bucket string `json:"bucket"`
 	// Region is a region where the bucket is located.
-	Region string `json:"region"`
+	Region string `json:"region,omitempty"`
```

Comment:
Starting from now we need to validate these fields on the BE side to be sure that provided configuration is valid and notify the end user if something is missing

Reply:
I agree

Reply:
Where can I find the validation PR for that?